bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/Root.scala

```diff
@@ -8,7 +8,7 @@ class Root(sem:Semester) extends SingleSelectNode[Semester, Instrument, Any](sem
   var title = "Select Instrument"
   var description = s"The following instruments are available for semester ${sem.year}${sem.half}. See the Gemini website for information on instrument capabilities and configuration options."
 
-  def choices = List(Alopeke, Flamingos2, GmosNorth, GmosSouth, Gnirs, Gpi, Graces, Gsaoi, Nifs, Niri, Visitor, Zorro)
+  def choices = List(Alopeke, Flamingos2, GmosNorth, GmosSouth, Gnirs, Gpi, Graces, Gsaoi, Igrins, Nifs, Niri, Visitor, Zorro)
```

Comment:
It was already this way before, I know, but let's go ahead and add the type to public methods?

```scala
def choices: List[Instrument] =
  List(Alopeke, ...)
```